js/search.js

```diff
@@ -1121,10 +1124,11 @@
 				if (id === 'skyattack') isViable = (toID(set.species) === 'hawlucha');
 				if (id === 'smackdown') isViable = (template.types.indexOf('ground') > 0);
 				if (id === 'smartstrike') isViable = (template.types.indexOf('steel') > 0 && moves.indexOf('ironhead') < 0);
-				if (id === 'solarbeam') isViable = (toID(set.abilities) in {drought:1, chlorophyll:1});
+				if (id === 'solarbeam' || id === 'solarblade') isViable = (toID(set.ability) in {desolateland:1, drought:1, chlorophyll:1});
 				if (id === 'stompingtantrum') isViable = ((moves.indexOf('earthquake') < 0 && moves.indexOf('drillrun') < 0) || (toID(set.ability) === 'toughclaws' && moves.indexOf('drillrun') < 0 && moves.indexOf('earthquake') < 0));
 				if (id === 'storedpower') isViable = (toID(set.species) in {necrozma:1, espeon:1, sigilyph:1});
 				if (id === 'stunspore') isViable = (moves.indexOf('thunderwave') < 0);
+				if (id === 'teleport') isViable = (this.gen > 7);
```

Comment:
Drypass is extremely rare... how many people actually use Teleport competitively?

Reply:
people use it as a slow pivoting move to get a guaranteed switch into a pokemon that a teammate checks

Reply:
I guess I won't judge. PS's standards for usable moves are supposed to be really low.